std/assembly/typedarray.ts

```diff
@@ -12,6 +12,15 @@ import {
   COMPARATOR
 } from "./internal/sort";
 
+@inline
+function clampToByte(value: i32): i32 {
+  if (ASC_OPTIMIZE_LEVEL < 2 || ASC_SHRINK_LEVEL > 0) {
+    return min(max(value, 0), 255);
```

Comment:
Would it also be an option to [optimize](https://graphics.stanford.edu/~seander/bithacks.html#IntegerMinOrMax) `min` and `max` instead? These are currently `select`s.

Reply:
This very special optimization only for `clamp(x, 0, 255)`. I tested bit twidding min/max approach with existing and it doesn't get significant benefit on Firefox, on Chrome this even slowdown on `~15%`.

Reply:
And I tested with different combinations. Compare single `min` and `max`, with compination `max(min(x, a), b)`, with constants and so on. In all cases existing approach faster

Reply:
Removed `@inline`